common/app/model/meta.scala

```diff
@@ -104,7 +104,7 @@ final case class Commercial(
     DfpAgent.isSponsored(tags.tags, Some(metadata.section), maybeEdition)
 
   def needsHighMerchandisingSlot(edition:Edition): Boolean = {
-    DfpAgent.isTargetedByHighMerch(metadata.adUnitSuffix,tags.tags,edition)
+    DfpAgent.isTargetedByHighMerch(metadata.adUnitSuffix,tags.tags,edition,metadata.url)
```

Comment:
Yes metadata.url does return a path rather than url, have made it clearer in tagSponsorship. Agree that metadata could do with updating to reflect path/url difference